### DISCUSSION TITLE
Include `value` in `hiddenSelectProps`?

### DISCUSSION THREAD

**Discussion by vincerubinetti** [chakra-ui/zag#1069] [General]
It appears that `hiddenSelectProps` ([for using getting a Zag select value in `FormData`](https://zagjs.com/components/react/select#usage-within-a-form)) includes a bunch of the needed props, but doesn't include the `value` which is needed.

I've done this for now:

```jsx
<select
  {...api.hiddenSelectProps}
  value={multi ? api.value : api.value[0]}
  onChange={() => null}
>
```

But it seems like it should just be included?

**Comment by segunadebayo** [comment]
Closing as outdated — the select API has been reworked in v1.0.
